libraries/cpp/rito.json

```diff
@@ -0,0 +1,19 @@
+{
+    "owner": "bartekprtc",
+    "repo": "rito",
+    "description": "Rito is a simple, crossplatform (Windows and Linux) C++20 library interfacing with Riot services (i.e. Riot REST API and League of Legends client).",
+    "language": "cpp",
+    "links": [
+        {
+            "name": "Source",
+            "url": "https://github.com/bartekprtc/rito"
```

Comment:
Botty adds GitHub source links automatically. This can be removed. :>

Reply:
removed, thanks!